Pod/Classes/SEGAmplitudeIntegration.m

```diff
@@ -159,11 +159,22 @@ - (void)screen:(SEGScreenPayload *)payload
 
 - (void)group:(SEGGroupPayload *)payload
 {
-    NSString *groupId = payload.groupId;
-    if (groupId) {
-        [self.amplitude setGroup:@"[Segment] Group" groupName:groupId];
-        SEGLog(@"[Amplitude setGroup:@'[Segment] Group' groupName:%@]", groupId);
+    NSString *groupName;
+    NSString *groupValue = payload.groupId;
+
+    NSString *groupTypeTrait = self.settings[@"groupTypeTrait"];
```

Comment:
What happens if `groupTypeTrait` is not set?

Reply:
It will default to the old behavior. Or are you saying to check if the value for this setting exists?

Reply:
Not quite. I was wondering what would happen at `payload.traits objectForKey:groupTypeTrait` if `groupTypeTrait` is null here.

Reply:
It will just default to the original behavior